go/cluster_management/README.md

```diff
@@ -22,8 +22,18 @@ In a terminal run the following commands:
 
 ```sh
 # Use the account credentials dedicated for golang
```

Comment:
This sentence is a bit weird for customers to read. Seems like it's left over from a previous iteration.